tech.json

```diff
@@ -522,31 +546,22 @@
               ],
               "note": [
                 "A mid-air Spring Ball jump that starts with a wall jump to gain more height.",
-                "It often relies on the momentum change when equipping or unequipping Spring Ball while morphed and moving horizontally."
-              ]
-            },
-            {
-              "name": "canDoubleSpringBallJumpMidAir",
-              "requires": [ "canTrickySpringBallJump" ],
-              "note": [
-                "Using a mid-air Spring Ball jump twice during a single jump to gain even more height.",
-                "This consists of a tight variant of mid-air Spring Ball jump, then turning off Spring Ball, then a second mid-air Spring Ball jump all while still climbing upwards.",
-                "This is typically used underwater with HiJump, but it is technically possible in lava without HiJump or to perform more than two Spring Ball jumps in acid."
+                "It often benefits from the momentum change when equipping Spring Ball while morphed and moving horizontally after a WallJump."
               ]
             },
-            {
-              "name": "canUnderwaterBombIntoSpringBallJump",
-              "requires": [ 
-                "canTrickySpringBallJump",
-                "h_canJumpIntoIBJ"
-              ],
-              "note": [
-                "Use the tiny amount of height gain from an underwater Bomb explosion to re-equip Springball and perform a Springball jump.",
-                "This is most often used to get a second SpringBall jump when underwater without HiJump.",
-                "Bombs are required since Power Bombs will prevent pausing."
-              ],
-              "devNote": "Similar to an underwater walljump into springballjump, but that has no applications yet since double springballjump is easier for similar requirements."
-            }
+          {
+            "name": "canSpringFling",
+            "requires": [
+              "canDisableEquipment",
+              "SpringBall"
```

Comment:
Does this require morph? (if not, I need to re-review its uses)

Reply:
Right.  I can get those helpers added.

Reply:
All the springballjump strats had a morph requirement already, so there was maybe one springfling which was missing morph.
There is a lot of potential as an artificial morph trick.  We can look into that more later.